src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportCreateParquetFilesStep.java

```diff
@@ -36,7 +45,22 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
     List<String> paths =
         bigQueryPdao.exportTableToParquet(snapshot, exportBucket, context.getFlightId());
 
-    workingMap.put(SnapshotWorkingMapKeys.SNAPSHOT_EXPORT_PARQUET_PATHS, paths);
+    Map<String, List<String>> tablesToPaths =
+        paths.stream()
+            .map(
+                path -> {
+                  String tableName = path.split("/")[4];
```

Comment:
Can we guarantee 4 slashes?

Reply:
We can! It will always be `gs://<bucket>/<flightId>/<tableName>/<parquetFiles*>`